ui/components/multichain/network-manager/components/default-networks/default-networks.tsx

```diff
@@ -77,6 +81,31 @@ const DefaultNetworks = memo(() => {
 
   const isEvmNetworkSelected = useSelector(getMultichainIsEvm);
 
+  const isMultichainAccountsState2Enabled = useSelector(
+    getIsMultichainAccountsState2Enabled,
+  );
+
+  // extract the evm account of the selected account group
+  const evmAccountGroup = useSelector((state) =>
+    getInternalAccountBySelectedAccountGroupAndCaip(state, EthScope.Eoa),
+  );
+
+  // extract the solana account of the selected account group
+  const solAccountGroup = useSelector((state) =>
+    getInternalAccountBySelectedAccountGroupAndCaip(
+      state,
+      'solana:5eykt4UsFv8P8NJdTREpY1vzqKqZKvdp',
```

Comment:
Do we have these network identifiers stored somewhere as a constant? If not, should we?